plugin/core/views.py

```diff
@@ -1002,6 +1006,15 @@ def format_completion(
     return completion
 
 
+def get_insert_replace_support_html(item: CompletionItem) -> Optional[str]:
+    text_edit = item.get('textEdit')
+    if text_edit and 'insert' in text_edit and 'replace' in text_edit:
+        insert_mode = userprefs().completion_insert_mode
+        oposite_insert_mode = 'replace' if insert_mode == 'insert' else 'insert'
+        return '<code>⇧ + ↵</code> to {}'.format(oposite_insert_mode)
```

Comment:
Not ideal if someone customizes the key binding...
Unless we don't really intend people to customize it but then we should bury keybinding at the bottom of keybindings settings.